arlas-core/src/main/java/io/arlas/server/app/ArlasBaseConfiguration.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.app;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.smoketurner.dropwizard.zipkin.ZipkinFactory;
+import io.arlas.server.exceptions.ArlasConfigurationException;
+import io.dropwizard.Configuration;
+import io.federecio.dropwizard.swagger.SwaggerBundleConfiguration;
+
+public class ArlasBaseConfiguration extends Configuration {
+
```

Comment:
We could add this part of conf, present in arlas-persistence basecode
```
@NotNull
    @JsonProperty("arlas_cors")
    public ArlasCorsConfiguration arlasCorsConfiguration;
```

Reply:
I'll do that in another PR when I have assessed the impact on the other microservices.